### PR TITLE
Ctcp 2929

### PR DESCRIPTION
**Pull request by srinivasulumunagalahmrc**

### CODE REVIEW

File: app/config/FrontendAppConfig.scala [hunk 1]
```diff
@@ -109,6 +110,13 @@ class FrontendAppConfig @Inject() (configuration: Configuration, phase5Switch: P
       s"$departureFrontendUrl/local-reference-number"
     }
 
+  val declareArrivalStartWithMRNUrl: String =
```
Comment (Reece-Carruthers) [inline app/config/FrontendAppConfig.scala:113]: There is a method above which already does this

  val declareArrivalNotificationStartUrl: String =
    if (phase5Switch.Arrivals.enabled) {
      declareArrivalNotificationUrlBase
    } else {
      s"$declareArrivalNotificationUrlBase/movement-reference-number"
    }

File: app/views/arrival/P5/ReviewUnloadingRemarkErrorsP5View.scala.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+        <p id="paragraph-1-prefix" class="govuk-body">@reviewUnloadingRemarkP5ViewModel.paragraph1Prefix</p>
+        <p id="paragraph-1-suffix" class="govuk-body">@reviewUnloadingRemarkP5ViewModel.paragraph1Suffix</p>
+
+        @answerSections(reviewUnloadingRemarkP5ViewModel.sections)
+
+        <p id="paragraph-2" class="govuk-body">@reviewUnloadingRemarkP5ViewModel.paragraph2Prefix <a id="helpdesk-link" class="govuk-link" target="_blank" href=@config.nctsEnquiriesUrl>@reviewUnloadingRemarkP5ViewModel.paragraph2Link</a> @reviewUnloadingRemarkP5ViewModel.paragraph2Suffix</p>
+
+        <p class="govuk-body">
+            <a id="arrival-link" class="govuk-link" href="@config.declareArrivalStartWithMRNUrl">@reviewUnloadingRemarkP5ViewModel.hyperlink</a>
```
Comment (Reece-Carruthers) [inline app/views/arrival/P5/ReviewUnloadingRemarkErrorsP5View.scala.html:48]: Change to @config.declareArrivalNotificationStartUrl

File: test/controllers/testOnly/ReviewUnloadingRemarkErrorsP5ControllerSpec.scala [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    reset(mockRejectionMessageActionProvider)
+  }
+
+  override def guiceApplicationBuilder(): GuiceApplicationBuilder =
+    super
+      .guiceApplicationBuilder()
+      .overrides(bind[ReviewUnloadingRemarkErrorsP5ViewModelProvider].toInstance(mockReviewUnloadingRemarkErrorMessageP5ViewModelProvider))
+      .overrides(bind[ArrivalP5MessageService].toInstance(mockArrivalP5MessageService))
+
+  "RejectionUnloadingRemarkP5Controller" - {
```
Comment (Reece-Carruthers) [inline test/controllers/testOnly/ReviewUnloadingRemarkErrorsP5ControllerSpec.scala:64]: minor change: RejectionUnloadingRemarkP5Controller ----> ReviewUnloadingRemarkErrorsP5ControllerSpec

File: test/views/arrival/P5/ReviewUnloadingRemarkErrorsP5ViewSpec.scala [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  behave like pageWithLink(
+    "helpdesk-link",
+    "New Computerised Transit System helpdesk",
+    frontendAppConfig.nctsEnquiriesUrl
+  )
+
+  behave like pageWithLink(
+    "arrival-link",
+    "View arrival notifications",
+    frontendAppConfig.declareArrivalStartWithMRNUrl
```
Comment (Reece-Carruthers) [inline test/views/arrival/P5/ReviewUnloadingRemarkErrorsP5ViewSpec.scala:99]: Same thing as the other comments change to @config.declareArrivalNotificationStartUrl

**Review by Reece-Carruthers — DISMISSED** [review]

**Review by kenneth-rayner — APPROVED** [review]

**Review by kenneth-rayner — APPROVED** [review]

**Review by vajarova — APPROVED** [review]

**Pull request merged**